_schemaV2.graphql

```diff
@@ -3172,6 +3172,44 @@ type BidIncrementsFormatted {
 # exceed the size of a 32-bit integer, it's encoded as a string.
 scalar BigInt
 
+type BulkUpdatePartnerArtworksMutationFailure {
+  mutationError: GravityMutationError
+}
+
+input BulkUpdatePartnerArtworksMutationInput {
+  # Whether Artsy domestic shipping should be enabled
+  artsyShippingDomestic: Boolean
+
+  # Whether Artsy international shipping should be enabled
+  artsyShippingInternational: Boolean
+  clientMutationId: String
+
+  # ID of the partner
+  id: String!
+
+  # The partner location ID to assign
+  location: String
+}
+
+type BulkUpdatePartnerArtworksMutationPayload {
+  bulkUpdatePartnerArtworksOrError: BulkUpdatePartnerArtworksMutationType
+  clientMutationId: String
+}
+
+type BulkUpdatePartnerArtworksMutationSuccess {
+  skippedPartnerArtworks: BulkUpdatePartnerArtworksResponse
+  updatedPartnerArtworks: BulkUpdatePartnerArtworksResponse
+}
+
+union BulkUpdatePartnerArtworksMutationType =
+    BulkUpdatePartnerArtworksMutationFailure
+  | BulkUpdatePartnerArtworksMutationSuccess
+
+type BulkUpdatePartnerArtworksResponse {
+  count: Int
+  ids: [String]
+}
```

Comment:
Shouldn't `errors` also be here? It is a part of the response and we seem to be using it in the `formattedReturn`